pkg/restapi/vc/operation/operations.go

```diff
@@ -1690,7 +1690,7 @@ func (o *Operation) parseAndVerifyVP(vpBytes []byte) (*verifiable.Presentation,
 			return nil, err
 		}
 		// verify if the credential in vp is valid
-		_, err = o.parseAndVerifyVCStrictMode(vcBytes)
+		err = o.validateCredentialProof(vcBytes, nil)
```

Comment:
I assume this is also in strict mode?

Reply:
Yes.

Reply:
this function internally calls parseAndVerifyVCStrictMode + proofPurpose validation.